src/c++/perf_analyzer/genai-perf/genai_perf/parser.py

```diff
@@ -349,10 +355,9 @@ def parse_args():
         passthrough_index = len(argv)
 
     args = parser.parse_args(argv[1:passthrough_index])
-    _check_conditional_args(parser, args)
+    args = _check_conditional_args(parser, args)
```

Comment:
The _check_conditional_args method now modifies the args object.
Can we update the type suggestion?

Reply:
Thanks for the feedback. Done!